static/app/utils/analytics/profilingAnalyticsEvents.tsx

```diff
@@ -0,0 +1,15 @@
+type ProfilingPrefix = 'profiling';
+type ProfilingViews =
+  | 'landing'
+  | 'transaction_summary'
+  | 'profile_summary'
+  | 'flamegraph';
+
+type EventKey = `${ProfilingPrefix}_views.${ProfilingViews}`;
+
+export const profilingEventMap: Record<EventKey, string> = {
+  'profiling_views.landing': 'Profiling views: Landing',
```

Comment:
@JonasBa Nit: Can we call it `Profiling Views`?

Reply:
Ah yes, 100%